src/components/Book.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+const mapStateToProps = state => ({
+  books: state.books,
+});
+
+const ConnectedBook = ({ books }) => (
+  <tbody>
+    {
+      books.map(book => {
+        const { id, title, category } = book;
+        return (
+          <tr key={id + title + category}>
+            <td key={id}>{ id }</td>
+            <td key={id + title}>{ title }</td>
+            <td key={id + category}>{ category }</td>
+          </tr>
+        );
+      })
+    }
+  </tbody>
+);
+
+ConnectedBook.propTypes = {
+  books: PropTypes.arrayOf(PropTypes.object),
+};
+
+ConnectedBook.defaultProps = {
+  books: [],
+};
+
+const Book = connect(mapStateToProps)(ConnectedBook);
+
+export default Book;
```

Comment:
- [ ] This component should represent only one book. The logic you put here, including `mapStateToProps` should be put in your `BookList` component, which should pass each `book` as a prop to this component. Don't get me wrong, the logic here is right, it's just a matter of putting almost all of the code here in another component, except for the JSX that's being returned in your `map` function :smile: